plugins/modules/ios_banner.py

```diff
@@ -108,11 +108,14 @@ def map_obj_to_commands(updates, module):
     if state == "absent" and "text" in have.keys() and have["text"]:
         commands.append("no banner %s" % module.params["banner"])
     elif state == "present":
+        if have.get("text") and want.get("text"):
+            if "\n" in have.get("text") and "\\n" not in want.get("text"):
+                have["text"] = have.get("text").rstrip("\n")
         if want["text"] and (want["text"] != have.get("text")):
             banner_cmd = "banner %s" % module.params["banner"]
-            banner_cmd += " @\n"
+            banner_cmd += " %\n"
```

Comment:
we gonna have a similar issue when we will be having % in the banner.
Could we have a different approach where we can decide special identifier at runtime?